README.md

```diff
@@ -16,7 +16,7 @@
 <h2 align="center">Install</h2>
 
 ```bash
-npm i -g|-D postcss-cli
+npm i -g|-D postcss postcss-cli
```

Comment:
I think it will be better to promote `-D` only and `npx postcss` call to run PostCSS.

It could be hard to use `--global` with `peerDependencies`.

Reply:
Good point.